src/Native/System.Net.Security.Native/pal_gssapi.h

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+#include "pal_types.h"
+
+#ifdef  GSSFW
+#include <GSS/GSS.h>
+#else
+#include <gssapi/gssapi_ext.h>
+#include <gssapi/gssapi_krb5.h>
+#endif
```

Comment:
Rather than using a single large `#if GSSFW`, please use smaller conditionals with CMake tests.  Then the resulting `#if` in the code is effectively self-documenting.